packages/core/src/components/sections/RegionBar/RegionBar.tsx

```diff
@@ -28,13 +29,17 @@ type RegionBarSectionProps = {
 }
 
 function RegionBarSection({
-  className = 'display-mobile',
+  className = '',
   ...otherProps
 }: RegionBarSectionProps) {
+  const { isMobile } = useScreenResize()
+
   return (
-    <Section className={`${styles.section} section-region-bar ${className}`}>
-      <RegionBar {...otherProps} />
-    </Section>
+    isMobile && (
+      <Section className={`${styles.section} section-region-bar ${className}`}>
```

Comment:
```suggestion
      <Section className={`${styles.section} section-region-bar`}>
```

we can remove if you agree with https://github.com/vtex/faststore/pull/2487/files#r1796232783